src/LaunchDarkly/ApcuLDDFeatureRequester.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
What's the intention for having this class vs ApcLDDFeatureRequester? Their phpdoc is the same. Adding use cases and examples to the Readme would also be great.